docs/mms_on_fargate.md

```diff
@@ -124,7 +124,11 @@ The healthcheck portion of your container configuration should look like the ima
 
 After configuring the health-checks, you can go onto configuring the environment, with the entry point that we have discussed earlier:
 
-![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/entrypoint.png)
+```bash
+mxnet-model-server, --mms-config, /mxnet_model_server/mms_app_cpu.conf
+```
+
+![](https://s3.amazonaws.com/mxnet-model-server/mms-github-docs/MMS+with+Fargate+Article/add+container+entry+point.png)
```

Comment:
Does this work? You need a keyword saying "start" right? I don't see it in the png file.